addons/mail/static/src/bugfix/bugfix.js

```diff
@@ -9,6 +9,44 @@ odoo.define('mail/static/src/bugfix/bugfix.js', function (require) {
 
 });
 
+// Should be moved to its own file in master.
```

Comment:
Can you please put version number instead of just "master"?
Master is always changing and we don't know which at which version during forward-port we must re-arrange the commit. That's why specify the exact Odoo version number of stuffs in bugfix files.

Reply:
Same as in the back button PR, I still don't understand this comment. We will always move it in whatever is the current master, we can never create a new file in stable. It's always at master at the time of forward-port.

Reply:
The file is created in master at some point, but eventually 2 different stable versions will differ with content in bugfix files in one and proper new files in other. If the comment keeps mentioning just master, we don't know at which version we have to re-arrange the commit.

That's why I propose to provide in comment the version that has the file, so that we know at which version we must eventually re-arrange the commits, and it's not necessarily master in the future.

Reply:
So you mean having "this is introduced in 14.0" in the comment?

Reply:
I'm thinking as you originally said some time ago that this comment is just useless. Every time in bugfix file the result will be the same: move to own file in master.

Reply:
> I'm thinking as you originally said some time ago that this comment is just useless. Every time in bugfix file the result will be the same: move to own file in master.

I'm worried of the habit to do many fixes in stable in earliest version + forward-port + r-plus in master when no conflict. I wouldn't be surprised by mistake of doing changes in bugfix files and not doing the change at the right version (or ever moving the changes to proper file in master at the fw chain of this PR). My suggestion to put version of file change in comment won't prevent any mistake, but I think it would help us know which fw command `up to` to set and make proper changes at the right versions, so much less changes to make mistakes.

This problem wouldn't happen if we opt for the reverse strategy of backward-porting, but that's not the Odoo way and manually doing it ourselves is tedious without tools like bots.

Reply:
It's not a big deal to forget to do the clean up at the end of the fw if there is no conflict.
We can always clean master later, there is no must-do clean time.

Not doing change in the right version is a bit more annoying but also not critical, as we can always backport manually if need be. And I don't see how changing this comment could prevent people from targeting wrong version. Especially since this mistake is much more likely to happen in standard code than in bugfix code, by pure statistical partition of code.

Most of the time there is no "up to", it's always full chain until master. The only non backport case for using "up to" is when there is a fix on a part of the code that no longer exists or has been heavily rewritten. In this case I totally agree with you, comment should not be "move to its own file in master" but "either move to XXX proper file in version YYY" (if said XXX file exists, or simply "do nothing", in other words, delete the bugfix past a certain version). Those are specific case, in all cases so far we just want own file in master.

So now I understand what you mean, but it only makes sense when we know "target file" already exists in a certain version.

Reply:
ok